src/stac_utils/reach.py

```diff
@@ -33,9 +33,10 @@ def refresh_auth(self, response: requests.Response):
             "username": self.api_user,
             "password": self.api_password,
         }
+        actual_base_url = "https://api.reach.vote"
```

Comment:
Just to follow the style of the rest of the class, could you move this up to the same level as `base_url`? Then you can call `self.actual_base_url` on line 39 instead